packages/protocol/contracts/governance/Election.sol

```diff
@@ -887,41 +895,25 @@ contract Election is
     return electedValidators;
   }
 
-  /**
-   * @notice Runs a round of the D'Hondt algorithm.
-   * @param electionGroups The addresses of the validator groups in the election.
-   * @param numMembers The number of members in each group.
-   * @param numMembersElected The number of members elected in each group up to this point.
-   * @dev See https://en.wikipedia.org/wiki/D%27Hondt_method#Allocation for more information.
-   * @return Whether or not a group elected a member, and the index of the group if so.
-   */
-  function dHondt(
-    address[] memory electionGroups,
-    uint256[] memory numMembers,
-    uint256 totalNumMembersElected,
-    uint256[] memory numMembersElected
-  ) private view returns (uint256, bool) {
-    bool memberElected = false;
-    uint256 groupIndex = 0;
-    FixidityLib.Fraction memory maxN = FixidityLib.wrap(0);
-    for (uint256 i = 0; i < electionGroups.length; i = i.add(1)) {
-      if (i > totalNumMembersElected) {
-        break;
+  // Top element dirty, fix heap
+  function heapify(uint256[] memory keys, FixidityLib.Fraction[] memory values) internal pure {
```

Comment:
To test this, consider creating an additional `HeapifyTest` contract that inherits from `Election`, exposes a public interface to `heapify`, generates `n` random test cases, and compares each of them against a trusted js implementation of heapify.

This is a pattern that's proven useful in testing some of the sorted linked list libraries.